File: app/src/main/java/com/csl/studio/photoshare/app/model/UserAttribute.java
package com.csl.studio.photoshare.app.model;


import java.util.HashMap;
import java.util.Map;

public class User {

    public String name = "";
    public String email = "";

    public Map<String, Object> toMap() {
        HashMap<String, Object> maps = new HashMap<>();
        maps.put("name", this.name);
        maps.put("email", this.email);
        return maps;
    }

}

File: app/src/main/java/com/csl/studio/photoshare/app/model/UserAttribute.kt
package com.csl.studio.photoshare.app.model


import java.util.HashMap

class UserAttribute {

    var name = ""
    var email = ""

    fun toMap(): Map<String, Any> {
        val maps = HashMap<String, Any>()
        maps.put("name", this.name)
        maps.put("email", this.email)
        return maps
    }

}
